extensions/panache/mongodb-panache/runtime/src/main/java/io/quarkus/mongodb/panache/PanacheMongoRepositoryBase.java

```diff
@@ -702,6 +702,17 @@ public default long deleteAll() {
         throw MongoOperations.implementationInjectionMissing();
     }
 
+    /**
+     * Delete an entity of this type by ID.
+     *
+     * @param id the ID of the entity to delete.
+     * @return false if the entity was not deleted (not found).
+     */
+    @GenerateBridge
+    public default boolean deleteById(Object id) {
```

Comment:
Same comment for `Id`

Reply:
It's not as easy as it looks, for `PanacheEntity` and for the repository pattern we don't know the name of the ID field so we need to retrieve it somehow ...

Reply:
done